### ISSUE TITLE
PassManager and FlowController section

### ISSUE THREAD

**Issue by 1ucian0** [Qiskit/documentation#891]
### URL, if applicable

~In tutorial section~  
~https://learning.quantum.ibm.com/catalog/tutorials~
In Transpile section https://docs.quantum.ibm.com/transpile

### Describe the new content you are requesting.

How to build a `PassManger` with different `FlowController`s and `property_set`. This includes differences between Analysis and Transformation passes.

Content Creator: @pedrorrivero

### If this new content request is accepted, do you want to write the content?

I only want to review the material when it's finished

**Comment by Eric-Arellano** [comment]
@1ucian0 by tutorial section, do you mean https://learning.quantum.ibm.com/catalog/tutorials?

**Comment by 1ucian0** [comment]
yeap, sounds good

**Comment by abbycross** [comment]
@kaelynj is working on another piece of work in the Transpile section (#1046) and can fold this in. Kaelyn, I can't recall from our meeting where we landed with it, but do you also need assistance from @pedrorrivero?

**Comment by kaelynj** [comment]
> @kaelynj is working on another piece of work in the Transpile section (#1046) and can fold this in. Kaelyn, I can't recall from our meeting where we landed with it, but do you also need assistance from @pedrorrivero?

I think I should be alright working on this myself.  I'll reach out to him if I end up needing help.

**Comment by abbycross** [comment]
This might be done already - @kaelynj do you agree? Is `FlowController` still missing?

**Comment by kaelynj** [comment]
Yeah `FlowController` is missing. We have an explanation of the difference b/w `Analysis` and `Transformation` passes in https://docs.quantum.ibm.com/guides/custom-transpiler-pass but the only page on building a custom `PassManager` focuses on dynamical decoupling: https://docs.quantum.ibm.com/guides/dynamical-decoupling-pass-manager

**Comment by kaelynj** [comment]
This issue has gone stale and no longer considers the current state of docs. I'll plan on reassessing this in the future if we need to make improvements to the transpiler pages.
